functions/error-reporting.ts

```diff
@@ -24,6 +25,15 @@ export const handler: Handler = async ({ body }) => {
       user,
     } = JSON.parse(body)
     Bugsnag.notify({ name, message, stack, cause }, (event) => {
+      event.app = {
+        releaseStage: 'production',
+        version: cliVersion,
+        type: 'netlify-cli',
```

Comment:
Instead of setting this manually should we instead use the config in `Bugsnag.start({})`
https://docs.bugsnag.com/platforms/javascript/configuration-options/#appversion

Reply:
we cannot set the CLI version on bugsnag start as this is a long-running function where multiple different CLI versions are reporting to